tests/lint/cpplint.sh

```diff
@@ -17,8 +17,9 @@
 # under the License.
 
 
-python3 3rdparty/dmlc-core/scripts/lint.py vta cpp vta/include vta/src
-python3 3rdparty/dmlc-core/scripts/lint.py tvm cpp \
+echo "Running 2 cpplints (VTA and TVM)..."
+python3 3rdparty/dmlc-core/scripts/lint.py --quiet vta cpp vta/include vta/src
+python3 3rdparty/dmlc-core/scripts/lint.py --quiet tvm cpp \
```

Comment:
These take kind of a while so it'd be nice to mention them separately

```suggestion
echo "Running VTA cpplint"
python3 3rdparty/dmlc-core/scripts/lint.py --quiet vta cpp vta/include vta/src
echo "Running TVM cpplint"
python3 3rdparty/dmlc-core/scripts/lint.py --quiet tvm cpp \
```